### ISSUE TITLE
luci-app-nut, Issues with Material theme

### ISSUE THREAD

**Issue by arrmo** [openwrt/luci#3942]
Hi,

Trying to find where the files are on the router itself, to try to fix this, but in the meantime ... hopefully the pics below demonstrate the issue. It's specific to the material theme, and for nut-monitor.

1) Bootstrap
![image](https://user-images.githubusercontent.com/8756384/79812097-7d7fe980-833d-11ea-87bb-4bdb80e35e90.png)

2) Material
![image](https://user-images.githubusercontent.com/8756384/79812107-840e6100-833d-11ea-9308-7f490834bfbe.png)

Basically, the select box needs to be taller - it sort of works right now, but really messes with selection. The list "leaks" outside the box (when trying to select).

Thanks!

**Comment by hnyman** [comment]
Material theme is pretty much unmaintained.

**Comment by arrmo** [comment]
Ahh, OK - gotcha. I thought it was sort of the "newer" one, but seems that's not the case. I'll try to dig, thanks.

**Comment by hnyman** [comment]
> I thought it was sort of the "newer" one

Well, that depends on your perspective.

When I started with OpenWrt ten years ago, there was just "openwrt theme", and somewhat later in 2012 came the current Bootstrap theme.

Material was introduced in 2015, but original author stopped contributing in 2017, so the theme is decaying.

Newly introduced well-maintained theme is luci-theme-openwrt-2020.

**Comment by arrmo** [comment]
Thanks for the info - much appreciated!

Will give that one a try.

**Comment by arrmo** [comment]
OK, added this to my build, really like it - thanks again!
